src/util/Unit.ts

```diff
@@ -1,5 +1,6 @@
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface UnitType {}
+export interface UnitType {
+	readonly type: unique symbol
```

Comment:
Suggestion - type be _nominal_UnitType (and marked @hidden + @deprecated) to follow nominal standards with roblox-ts.